### PR TITLE
Move timeline tool and statvar menu code to separate folders

### PR DESCRIPTION
**Pull request by shifucun**
This is to help for further refactor. No functional change in this PR.

### CODE REVIEW

File: static/webpack.config.js [hunk 1]
```diff
@@ -32,10 +32,10 @@ const config = {
     ],
     dev: [__dirname + "/js/dev.ts", __dirname + "/css/dev.scss"],
     timeline: [
-      __dirname + "/js/tools/timeline.ts",
+      __dirname + "/js/tools/timeline/timeline.ts",
       __dirname + "/css/timeline.scss",
     ],
-    timeline_bulk_download: [__dirname + "/js/tools/timeline_bulk_download.ts"],
+    timeline_bulk_download: [__dirname + "/js/tools/timeline/bulk_download.ts"],
```
Comment (beets) [inline static/webpack.config.js:38]: dont forget to also update the corresponding html file

**Review by beets — APPROVED** [review]
thanks for the refactor. it'll also be nice to eventually drop the middle "s" in "statsvar" to "statvar"

**Comment by beets** [comment]
will you be refactoring this to be more of a standalone component?

**Review by chejennifer — APPROVED** [review]
Thanks for refactoring!

File: static/webpack.config.js (hunk 1 above)
Comment (shifucun) [reply static/webpack.config.js:38]: Just tested the js name is based on the entry key "timeline_bulk_download" so this should be good.

**Comment by shifucun** [comment]
> will you be refactoring this to be more of a standalone component?

Yep. The statvar menu (or svg menu) will be a standalone component. I plan to take that out from current browser code.

**Comment by shifucun** [comment]
> thanks for the refactor. it'll also be nice to eventually drop the middle "s" in "statsvar" to "statvar"

sg

**Pull request merged**